src/Controls/src/Core/LegacyLayouts/Layout.cs

```diff
@@ -152,24 +156,16 @@ public Thickness Padding
 		/// </value>
 		public bool CascadeInputTransparent
 		{
-			get => (bool)GetValue(CascadeInputTransparentProperty);
-			set => SetValue(CascadeInputTransparentProperty, value);
+			get => (bool)GetValue(InputTransparentContainerElement.CascadeInputTransparentProperty);
+			set => SetValue(InputTransparentContainerElement.CascadeInputTransparentProperty, value);
```

Comment:
This does not need to change since the properties are still in this type.


```suggestion
			get => (bool)GetValue(CascadeInputTransparentProperty);
			set => SetValue(CascadeInputTransparentProperty, value);
```